src/simtools/db/db_handler.py

```diff
@@ -263,99 +292,106 @@ def get_collection(self, db_name, collection_name):
         db_name = self._get_db_name(db_name)
         return DatabaseHandler.db_client[db_name][collection_name]
 
-    def export_file_db(self, db_name, dest, file_name):
+    def get_collections(self, db_name=None, model_collections_only=False):
         """
-        Get file from the DB and write to disk.
+        List of collections in the DB.
 
         Parameters
         ----------
         db_name: str
-            Name of the DB to search in.
-        dest: str or Path
-            Location where to write the file to.
-        file_name: str
-            Name of the file to get.
+            Database name.
+        model_collections_only: bool
+            If True, only return model collections (i.e. exclude fs.files, fs.chunks, metadata)
 
         Returns
         -------
-        file_id: GridOut._id
-            the database ID the file was assigned when it was inserted to the DB.
-
-        Raises
-        ------
-        FileNotFoundError
-            If the desired file is not found.
+        list
+            List of collection names
 
         """
-        db_name = self._get_db_name(db_name)
-
-        self._logger.debug(f"Getting {file_name} from {db_name} and writing it to {dest}")
-        file_path_instance = self._get_file_mongo_db(db_name, file_name)
-        self._write_file_from_mongo_to_disk(db_name, dest, file_path_instance)
-        return file_path_instance._id  # pylint: disable=protected-access;
+        db_name = db_name or self._get_db_name()
+        if db_name not in self.list_of_collections:
+            self.list_of_collections[db_name] = DatabaseHandler.db_client[
+                db_name
+            ].list_collection_names()
+        collections = self.list_of_collections[db_name]
+        if model_collections_only:
+            return [
+                collection
+                for collection in collections
+                if not collection.startswith("fs.") and collection != "metadata"
```

Comment:
Will we still have a metadata collection after the restructuring?

Reply:
We don't have a metadata collection since PR #1166. Removed above reference to "metadata"